_posts/2020-09-02-cockpit-227.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 227
+author: pitti
+date: 2020-09-02
+tags: cockpit linux
+slug: cockpit-227
+category: release
+summary: Cockpit with VM filtering and PatternFly 4 design changes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 227.
+
+### Machines: VM list filtering
+
+The Virtual Machine overview page can now be filtered by name and/or by state.
+
+![VM filtering](/images/machines-vm-list-filter.png)
+
+### Continued PatternFly 4 migration
+
+The Networking, Firewall, Services, and Storage pages use more [PatternFly 4](https://www.patternfly.org/v4/) elements, which continuously improves accessibility and mobile mode.
+
+### Stop releasing to Fedora 31
```

Comment:
```suggestion
### Discontinue Fedora 31 releases
```

Reply:
```suggestion
### Stop Fedora 31 releases
```

Reply:
It feels slightly weird as a heading (perhaps due to the "to"), so I provided two alternate suggestions. But they might also be a bit weird. Feel free to take either suggestion or keep it as-is. It's your preference.